src/components/SetFee.vue

```diff
@@ -0,0 +1,389 @@
+          low: this.drValues ? drt_low : vtt_low,
+          medium: this.drValues ? drt_medium : vtt_medium,
+          high: this.drValues ? drt_high : vtt_high,
+          opulent: this.drValues ? drt_opulent : vtt_opulent,
+        }
+      } else {
+        return {}
+      }
+    },
+    estimationOptions() {
+      if (this.estimatedTransactions && this.formatedFeeEstimationReport) {
+        const result = FEE_TRAITS.reduce((acc, trait) => {
+          acc.push({
+            label: trait,
+            report: this.formatedFeeEstimationReport[trait],
+            transaction: this.estimatedTransactions
+              ? this.estimatedTransactions[trait]
+              : {},
+          })
+          return acc
+        }, [])
+        return [...result, { label: 'custom' }]
+      } else {
+        return []
+      }
+    },
+    feeType() {
+      return this.feeValues.isWeightedFee
+        ? {
+            key: 'weighted',
+            text: this.$t('weighted_fee'),
+          }
+        : {
+            key: 'absolute',
+            text: this.$t('absolute_fee'),
+          }
+    },
+  },
+  async mounted() {
+    await this.getFeeEstimationReport()
+    await this.getEstimatedTransactions()
+  },
+  methods: {
+    ...mapMutations({
+      clearError: 'clearError',
+      clearGeneratedTransaction: 'clearGeneratedTransaction',
+    }),
+    ...mapActions({
+      getFeeEstimationReport: 'getFeeEstimationReport',
+      createDataRequest: 'createDataRequest',
+      createVTT: 'createVTT',
+    }),
+    async getEstimatedTransactions() {
+      const txRequests = FEE_TRAITS.map(async trait => {
+        let transaction
+        if (this.drValues && this.formatedFeeEstimationReport[trait]) {
+          transaction = await this.createDataRequest({
+            parameters: {
+              ...this.drValues,
+              fee: this.formatedFeeEstimationReport[trait].priority,
+              feeType: this.feeType,
+            },
+            request: this.drValues.template.radRequest,
+          })
+        }
+        if (this.vttValues && this.formatedFeeEstimationReport[trait]) {
+          transaction = await this.createVTT({
+            ...this.vttValues,
+            fee: this.formatedFeeEstimationReport[trait].priority,
+            feeType: this.feeType,
+          })
+        }
+        if (
+          this.createDataRequestError &&
+          this.createDataRequestError.error !== 'Validation Error'
+        ) {
+          this.clearValues()
+        }
+        if (
+          this.createVTTError &&
+          this.createVTTError.error !== 'Validation Error'
+        ) {
+          this.clearValues()
+        }
+        return {
+          label: trait,
+          result: await transaction,
+        }
+      })
+      Promise.all(txRequests).then(result => {
+        this.estimatedTransactions = result.reduce((acc, tx) => {
+          acc[tx.label] = tx.result
+          return acc
+        }, {})
```

Comment:
Just in case you're a fan of one-liners:
```suggestion
        this.estimatedTransactions = result.reduce((acc, tx) => ({...acc, [tx.label]: tx.result}), {})
```